CogNative/backend/modules/audio_split.py

```diff
@@ -0,0 +1,26 @@
+from pydub import AudioSegment
+from pydub.silence import split_on_silence
+from colorama import Fore
+import os
+
+# user input for audio file + file existence check
+aud_file = input("Enter name of audio file: ")
+file_check = os.path.exists(f"Clone_Tests/Raw/{aud_file}.mp3")
+assert file_check, print(f"{Fore.YELLOW}Please move audio file to 'Clone_Tests/Raw/'")

```

Comment:
Might want to change extension to wav and adjust the filepath for future integration.